src/Pages/3DPrintingConsole/3D-console.css

```diff
@@ -22,7 +22,19 @@
   color: #61dafb;
 }
 
-#secondRow
-{
-  margin-left: 35px !important;
+.Console-3d-label-1 {
+  display: none;
 }
+
+@media only screen and (max-width: 1090px) {
+  .Console-3d-label {
+    display: none;
+  }
+  .Console-3d-label-1 {
+    display: block;
+    margin-top: 10px;
+  }
+  #Console-3d-buttons {
+    margin-top: 10px;
+  }
+}
```

Comment:
keep the newline at the bottom of this file. [this blog post](https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/) is helpful for explaining why

Reply:
this is not resolved right? just add an empty line at the end of the bracket.

Reply:
![image](https://user-images.githubusercontent.com/18486562/74484542-9cf62400-4e6d-11ea-9a45-9285a1b0faa0.png)
that little red arrow indicates "no new line at teh end of file"